controllers/datadogagent/feature/cspm/feature.go

```diff
@@ -94,6 +95,10 @@ func (f *cspmFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp feature.Req
 		// TODO add settings to configure f.createPSP
 		f.createSCC = v2alpha1.ShouldCreateSCC(dda, v2alpha1.NodeAgentComponentName)
 
+		if dda.Spec.Features.CSPM.HostBenchmarks != nil && *dda.Spec.Features.CSPM.HostBenchmarks.Enabled {
+			f.hostBenchmarksEnabled = true
+		}
```

Comment:
I’m wondering if it’s possible to have:
```go
dda.Spec.Features.CSPM.HostBenchmarks = &CSPMHostBenchmarksConfig{
  Enabled: nil,
}
```
at this point, in which case it would trigger a `nil`-pointer panic.

I think it would be safer so use `apiutils.BoolValue(…)`:
```suggestion
		if dda.Spec.Features.CSPM.HostBenchmarks != nil && apiutils.BoolValue(dda.Spec.Features.CSPM.HostBenchmarks.Enabled) {
			f.hostBenchmarksEnabled = true
		}
```